cmake/fficonfig.h.in

```diff
@@ -45,98 +45,98 @@
 
 /* Define to 1 if you have the `memcpy' function. */
-#define HAVE_MEMCPY 1
+#cmakedefine HAVE_MEMCPY 1
 
 /* Define to 1 if you have the <memory.h> header file. */
-#define HAVE_MEMORY_H 1
+#cmakedefine HAVE_MEMORY_H 1
 
 /* Define to 1 if you have the `mmap' function. */
-#define HAVE_MMAP 1
+#cmakedefine HAVE_MMAP 1
 
 /* Define if mmap with MAP_ANON(YMOUS) works. */
-#define HAVE_MMAP_ANON 1
+#cmakedefine HAVE_MMAP_ANON 1
 
 /* Define if mmap of /dev/zero works. */
-#define HAVE_MMAP_DEV_ZERO 1
+#cmakedefine HAVE_MMAP_DEV_ZERO 1
 
 /* Define if read-only mmap of a plain file works. */
-#define HAVE_MMAP_FILE 1
+#cmakedefine HAVE_MMAP_FILE 1
 
 /* Define if .eh_frame sections should be read-only. */
 /* #undef HAVE_RO_EH_FRAME */
 
 /* Define to 1 if you have the <stdint.h> header file. */
-#define HAVE_STDINT_H 1
+#cmakedefine HAVE_STDINT_H 1
 
 /* Define to 1 if you have the <stdlib.h> header file. */
-#define HAVE_STDLIB_H 1
+#cmakedefine HAVE_STDLIB_H 1
 
 /* Define to 1 if you have the <strings.h> header file. */
-#define HAVE_STRINGS_H 1
+#cmakedefine HAVE_STRINGS_H 1
 
 /* Define to 1 if you have the <string.h> header file. */
-#define HAVE_STRING_H 1
+#cmakedefine HAVE_STRING_H 1
 
 /* Define to 1 if you have the <sys/mman.h> header file. */
-#define HAVE_SYS_MMAN_H 1
+#cmakedefine HAVE_SYS_MMAN_H 1
 
 /* Define to 1 if you have the <sys/stat.h> header file. */
-#define HAVE_SYS_STAT_H 1
+#cmakedefine HAVE_SYS_STAT_H 1
 
 /* Define to 1 if you have the <sys/types.h> header file. */
-#define HAVE_SYS_TYPES_H 1
+#cmakedefine HAVE_SYS_TYPES_H 1
 
 /* Define to 1 if you have the <unistd.h> header file. */
-#define HAVE_UNISTD_H 1
+#cmakedefine HAVE_UNISTD_H 1
 
 /* Define to the sub-directory in which libtool stores uninstalled libraries.
    */
-#define LT_OBJDIR ".libs/"
+#cmakedefine LT_OBJDIR ".libs/"
 
 /* Define to 1 if your C compiler doesn't accept -c and -o together. */
 /* #undef NO_MINUS_C_MINUS_O */
 
 /* Name of package */
-#define PACKAGE "libffi"
+#cmakedefine PACKAGE "libffi"
 
 /* Define to the address where bug reports for this package should be sent. */
-#define PACKAGE_BUGREPORT "http://gcc.gnu.org/bugs.html"
+#cmakedefine PACKAGE_BUGREPORT "http://gcc.gnu.org/bugs.html"
 
 /* Define to the full name of this package. */
-#define PACKAGE_NAME "libffi"
+#cmakedefine PACKAGE_NAME "libffi"
 
 /* Define to the full name and version of this package. */
-#define PACKAGE_STRING "libffi 3.0.10rc0"
+#cmakedefine PACKAGE_STRING "libffi 3.0.10rc0"
 
 /* Define to the one symbol short name of this package. */
-#define PACKAGE_TARNAME "libffi"
+#cmakedefine PACKAGE_TARNAME "libffi"
 
 /* Define to the home page for this package. */
-#define PACKAGE_URL ""
+#cmakedefine PACKAGE_URL ""
 
 /* Define to the version of this package. */
-#define PACKAGE_VERSION "3.0.10rc0"
+#cmakedefine PACKAGE_VERSION "3.0.10rc0"
 
 /* The size of `double', as computed by sizeof. */
-#define SIZEOF_DOUBLE 8
+#cmakedefine SIZEOF_DOUBLE 8
```

Comment:
Test [here](https://github.com/python-cmake-buildsystem/python-cmake-buildsystem/blob/5b246c43d012647a194a5b50595dc5968caf13ce/cmake/ConfigureChecks.cmake#L586)

Should be changed to `#cmakedefine SIZEOF_DOUBLE @SIZEOF_DOUBLE@`